content/docs/replication/upgrade.md

```diff
@@ -27,11 +27,11 @@ To upgrade the CSM Replication sidecar that is installed along with the driver,
 
 ### PowerScale
 
-On PowerScale systems, an additional step is needed when upgrading from CSM v1.5 to CSM v1.6. Because the SyncIQ policy created on the target-side storage array is no longer used, it must be deleted for any existing replication groups after performing the upgrade to the CSM Replication sidecar and PowerScale CSI driver. These steps should be performed before the replication groups are used with the new version of the CSI driver. Until this step is performed, Replication Groups created on CSM v1.5 will display an UNKNOWN link state in CSM v1.6.
+On PowerScale systems, an additional step is needed when upgrading to CSM Replication v1.4.0 or later. Because the SyncIQ policy created on the target-side storage array is no longer used, it must be deleted for any existing `DellCSIReplicationGroup` objects after performing the upgrade to the CSM Replication sidecar and PowerScale CSI driver. These steps should be performed before the `DellCSIReplicationGroup` objects are used with the new version of the CSI driver. Until this step is performed, existing `DellCSIReplicationGroup` objects will display an UNKNOWN link state.
 
 1. Log in to the target PowerScale array. 
 2. Navigate to the `Data Protection > SyncIQ` page and select the `Policies` tab.
-3. Delete all disabled, target-side SyncIQ policies that are used for CSM Replication. Such policies will be distinguished by their names, of the format `<prefix>-<kubernetes namespace>-<IP of replication destination>-<RPO duration>`.
+3. Delete all disabled SyncIQ policies that are used for CSM Replication. Such policies will be distinguished by their names, of the format `<prefix>-<kubernetes namespace>-<IP of replication destination>-<RPO duration>`.
```

Comment:
Why did target-side be commented out? Is there no possibility of a policy that is disabled but not be deleted?

Reply:
I'm also curious; why was the target-side verbage removed here? Source-side SIQ policies may be disabled in failover state, for example.

Reply:
To avoid the confusion with Local Targets tab during this step. Previous steps clearly mention 'log in to **target** array' which I think is sufficient.

Reply:
Is there a way to clearly communicate to the consumer to safely identify their disabled SYNQ IQ policies and only delete those? I am worried if they DELETE ALL DISABLED SYNCIQ policies.